src/tests/ftest/rebuild/no_cap.yaml

```diff
@@ -10,6 +10,8 @@ timeout: 180
 server_config:
   engines_per_host: 2
   name: daos_server
+  env_vars:
+    - CRT_TIMEOUT=10
```

Comment:
Can we please add some comments above this value change to help somebody looking at the test why this change is being made. It seems like , this is a workaround, so a slightly detailed description why we need to do this and when it should be removed will be helpful.
Also, it will be nice if a new ticket is opened (if not already created) to solve the real problem as this is a workaround.